src/engine/driver/Driver.java

```diff
@@ -24,18 +24,19 @@ public class Driver {
 	 * @param args
 	 */
 	public static void main(String[] args) {
-		char delim = args[3].charAt(0);
-		String filePathIn = args[1];
-		String filePathOut = args[2];
+		char delim = args[2].charAt(0);
```

Comment:
Pretty sure it starts at args[1] might be a c thing, please check that.

Reply:
I had the outputs working when running debugging